scripts/setup-env-macos.sh

```diff
@@ -1,31 +1,26 @@
 #!/bin/bash
 set -x
 
-# Quick install script for mac
-
-# Install homebrew
+# Install Homebrew.
 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
 
-# Brew install all packages
+# Brew install all packages.
 brew upgrade
 
 brew install \
-	go \
-	libomp \
-	node \
-	git \
-	pyenv \
-	yarn
+  go \
+  libomp \
+  node \
+  git \
+  pyenv \
+  yarn
 
 brew cask install \
-	docker
+  docker
 
-# Install Python 3.6 and virtualenv
 pyenv install 3.6.9
-python3 -m pip install virtualenv
 
-# Set up go mono repo
-mkdir -p "$HOME/go"
-mkdir -p "$HOME/go/bin"
-mkdir -p "$HOME/go/pkg"
-mkdir -p "$HOME/go/src"
```

Comment:
@dzhu could you echo a tip here on ensuring that the go install's bin is on the developer's path?  something like `echo 'Please ensure that your Go installation bin is on your $PATH, e.g., by adding this to your shell startup script: export PATH=$PATH:$HOME/go/bin'`

I hit this snag in setting up my devenv and a pointer would have helped.  This definitely applies to Mac; not sure about other OSes.

I also think it's worth echoing this pointer in the [setup section of the Contributor's Guide](https://github.com/determined-ai/determined/blob/master/CONTRIBUTING.md#setting-up)